sensors/sensors.go

```diff
@@ -72,18 +72,16 @@ func NewLidar(
 		return Lidar{}, errors.Wrapf(err, "error getting lidar camera %v for slam service", cameraName)
 	}
 
-	// https://viam.atlassian.net/browse/RSDK-4306
-	// To be implemented once replay camera supports Properties
-	// // If there is a camera provided in the 'camera' field, we enforce that it supports PCD.
-	// properties, err := newLidar.Properties(ctx)
-	// if err != nil {
-	// 	return Lidar{}, errors.Wrapf(err, "error getting lidar camera properties %v for slam service", cameraName)
-	// }
+	// If there is a camera provided in the 'camera' field, we enforce that it supports PCD.
```

Comment:
One more thing: Could we change the name of this ticket/PR to reflect this change, which I think is the main change?

Reply:
Done: https://viam.atlassian.net/browse/RSDK-4306

Reply:
ty!